src/components/Widgets/SubscriptionsWidget.tsx

```diff
@@ -3,51 +3,49 @@ import OutlinedCalendarAltIcon from '@patternfly/react-icons/dist/js/icons/outli
 import useStatus from '../../hooks/useStatus';
 import { QueryClient, QueryClientProvider } from 'react-query';
 import './SubscriptionsWidget.scss';
-import {
-  EmptyState,
-  EmptyStateVariant,
-  EmptyStateIcon,
-  EmptyStateBody,
-  Stack,
-  StackItem,
-  Title,
-  Alert,
-  AlertVariant,
-  Gallery
-} from '@patternfly/react-core';
+import { EmptyState } from '@patternfly/react-core/dist/dynamic/components/EmptyState';
+import { EmptyStateVariant } from '@patternfly/react-core/dist/dynamic/components/EmptyState';
+import { EmptyStateIcon } from '@patternfly/react-core/dist/dynamic/components/EmptyState';
+import { EmptyStateBody } from '@patternfly/react-core/dist/dynamic/components/EmptyState';
+import { Stack } from '@patternfly/react-core/dist/dynamic/layouts/Stack';
+import { StackItem } from '@patternfly/react-core/dist/dynamic/layouts/Stack';
+import { Title } from '@patternfly/react-core/dist/dynamic/components/Title';
+import { Alert } from '@patternfly/react-core/dist/dynamic/components/Alert';
+import { AlertVariant } from '@patternfly/react-core/dist/dynamic/components/Alert';
+import { Gallery } from '@patternfly/react-core/dist/dynamic/layouts/Gallery';
 import { Link } from 'react-router-dom';
 import EmptyStateSubscriptionsIcon from './public/images/SubscriptionsWidgetEmptyStateIcon';
-import { Spinner } from '@patternfly/react-core';
-
+import { Spinner } from '@patternfly/react-core/dist/dynamic/components/Spinner';
 const queryClient = new QueryClient();
-
 const SubscriptionsWidget = () => {
   return (
     <QueryClientProvider client={queryClient}>
       <SubsWidget />
     </QueryClientProvider>
   );
 };
-
 const cardData = {
   active: {
     title: 'Active',
-    variant: AlertVariant.success
+    variant: AlertVariant.success,
+    filter: 'active'
   },
   expiringSoon: {
     title: 'Expiring soon',
-    variant: AlertVariant.warning
+    variant: AlertVariant.warning,
+    filter: 'expiringSoon'
   },
   expired: {
     title: 'Expired',
-    variant: AlertVariant.danger
+    variant: AlertVariant.danger,
+    filter: 'expired'
   },
   futureDated: {
     title: 'Future dated',
-    customIcon: <OutlinedCalendarAltIcon />
+    customIcon: <OutlinedCalendarAltIcon />,
+    filter: 'futureDated'
```

Comment:
Suggestion: It looks like the `filter` value is the same as the key in every case. We can just use the keys as the basis for filtering